### ISSUE TITLE
Заражение фороном одежды под ригом.

### ISSUE THREAD

**Issue by GodzerBoot** [TauCetiStation/TauCetiClassic#10440]
#### Подробное описание проблемы
Форон в атмосфере заражает одежду сквозь риг
#### Что должно было произойти
Риг должен защитить одежду под собой от форона
#### Что произошло на самом деле
Одежда обрела значок биохазарда
#### Как повторить
Войти в атмосферу форона в риге и одежде под ним
#### Дополнительная информация:
Issue reported from  Round ID: 56018 (Tau Ceti Classic (RU))
Testmerges: ```#8857 #9911 #10127 #10207 #10076 ```
Reporting client version: 514.1589

**Comment by Deahaka** [comment]
Вроде так и задумано. Там есть шанс 1% что форон просочится. Хз зачем

**Comment by kalazus** [comment]
Нет, там просто зерновая проверка которую пытались починить не так давно
